readthedocs/settings/base.py

```diff
@@ -333,132 +333,39 @@ def INSTALLED_APPS(self):  # noqa
     SILENCED_SYSTEM_CHECKS = ['fields.W342']
 
     # Logging
-    LOG_FORMAT = "[%(asctime)s] %(levelname)s [%(name)s:%(lineno)s] %(message)s"
+    LOG_FORMAT = '%(name)s[%(process)d]: %(levelname)s %(message)s [%(name)s:%(lineno)s]'
```

Comment:
Good call, I addressed this in a5389dd

I only added the datetime to the development settings, as we don't need to repeat this in production. I also used the format that would match the output of the http logging.